import_data/tasks.py

```diff
@@ -10,13 +11,69 @@
 import time
 from datetime import timedelta, datetime
 import requests
-from invoke import task
+from invoke import context, task
 from invoke.exceptions import Failure
 from utils.lock import FileLock
 
+cc_exec = concurrent.futures.ThreadPoolExecutor()
+
 logging.basicConfig(level=logging.INFO)
 
 
+class PrefixedStream:
+    """
+    Wrapper class around a stream adding a prefix to each writen line.
+    """
+    def __init__(self, src_stream, prefix):
+        self.prefix = prefix
+        self.src_stream = src_stream
+        self.buffer = ''
+
+    def write(self, data):
+        self.buffer += data
+        self.flush()
+
+    def flush(self):
+        lines = self.buffer.split('\n')
+
+        for line in lines[:-1]:
+            self.src_stream.write(
+                '\033[0;90m[{}]\033[0;0m {}\n'.format(self.prefix, line)
+            )
+
+        self.buffer = lines[-1]
+        self.src_stream.flush()
+
+
+class PrefixedContext(context.Context):
+    """
+    Wrapper class around a pyinvoke context adding a prefix to each lines
+    outputed by ctx.run(..).
+    """
+    def __init__(self, ctx, prefix):
+        self.__dict__ = ctx.__dict__.copy()
+        self.prefix = prefix
+
+    def run(self, *args, **kwargs):
+        return super().run(
+            *args,
+            **kwargs,
+            out_stream=PrefixedStream(sys.stdout, self.prefix),
+            err_stream=PrefixedStream(sys.stderr, self.prefix + ':ERR'),
+        )
+
+def format_stdout(fun):
+    """
+    Wrapper decorator around a task adding its name to each line outputed by
+    ctx.run(..).
+    """
+    def upgraded_fun(ctx, *args, **kwargs):
+        ctx = PrefixedContext(ctx, fun.__name__)
+        fun(ctx, *args, **kwargs)
+
+    upgraded_fun.__name__ = fun.__name__
```

Comment:
When defining this kind of decorator, it may be convenient to use [`functools.wraps`](https://docs.python.org/3.8/library/functools.html#functools.wraps) to preserve name, docstring, etc.